pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -668,6 +668,7 @@ enum ContainerState {
     CONTAINER_RUNNING = 1;
     CONTAINER_EXITED  = 2;
     CONTAINER_UNKNOWN = 3;
+    CONTAINER_OOMKILLED = 4;
```

Comment:
If this is the only place that needs the "OOMKilled" information, I'd say remove this. If kubernetes ever needs to explicitly rely on the "OOMKilled" reason, I think we should add an additional field in the container status.